plugin-maven/src/main/java/com/diffplug/spotless/maven/FileLocator.java

```diff
@@ -45,6 +45,11 @@ public File locateFile(String path) {
 			return null;
 		}
 
+		File testExists = new File(path);
+		if (testExists.exists() && testExists.isFile()) {
```

Comment:
Maybe rename `testExists` to `localFile`?

Reply:
Good call.